reposilite-plugins/javadoc-plugin/src/main/kotlin/com/reposilite/plugin/javadoc/JavadocPlugin.kt

```diff
@@ -0,0 +1,39 @@
+package com.reposilite.plugin.javadoc
+
+import com.reposilite.Reposilite
+import com.reposilite.maven.MavenFacade
+import com.reposilite.plugin.api.Facade
+import com.reposilite.plugin.api.Plugin
+import com.reposilite.plugin.api.ReposiliteInitializeEvent
+import com.reposilite.plugin.api.ReposilitePlugin
+import com.reposilite.web.api.RoutingSetupEvent
+import java.io.File
+
+@Plugin(name = "JavadocPlugin")
+class JavadocPlugin : ReposilitePlugin() {
+
+    private lateinit var reposilite: Reposilite
+    private lateinit var javadocFolder: File
```

Comment:
Keeping these values as lateinit fields is kinda bad practice and there is no point to do this, check next comments below.